wazo_plugind/schema.py

```diff
@@ -114,6 +120,7 @@ class OptionField(fields.Field):
 
     _options = {
         'git': fields.Nested(GitInstallOptionsSchema),
+        'archive': fields.Nested(ArchiveInstallOptionsSchema),
```

Comment:
Out of curiosity, does the `archive` option is intended to be merged in master? (i.e. part of the spec of plugind?)
Or it's only for a POC?

Reply:
Sorry, I should have created this as a draft, I'll change it. It's a test for provd-plugins. I was trying to test a few different ways the plugins could be built and distributed. Installing via git currently requires a separate repo per plugin, hence the test with sub directories and the archive option was if we need to `build` plugins in the CI still, creating and archive to download is easy and doesn't need to be committed in the repo. So far just for testing though.